fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -3771,3 +3771,55 @@ countries:
                 length: 3.0
                 issue_months: [5, 6, 7, 8]
                 start_year: 1983
+    zambia:
+        logo: Zambia_IRI_120x48.png
+        resolution: [0.05, 0.05]
+        center: [27, -15]
+        zoom: 6
+        rotation: 0
+        marker: [25, -17]
+        shapes:
+            - name: National
+              sql: select adm0_code as key, adm0_name as label, ST_AsBinary(the_geom) as the_geom
+                  from iridb.g2015_2012_0 where adm0_name = 'Zambia'
+              vuln_sql: select adm0_code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Zambia' group by 1, 2
```

Comment:
This presumably doesn't work, because there are no rows for Zambia in the fbf_vulnerability table. That will cease to be a problem once the vulnerability layer is removed, but it would be good to merge this sooner than that. See `ethiopia` for a dummy query you can use instead.

Reply:
The vulnerability layer has been removed, so now all we need to do is remove `vuln_sql`.